flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/ExecutionPartitionLifecycleTest.java

```diff
@@ -316,7 +317,10 @@ private JobVertex createNoOpJobVertex() {
         return jobVertex;
     }
 
-    private static class TestingShuffleMaster implements ShuffleMaster<ShuffleDescriptor> {
+    private static class TestingShuffleMaster
```

Comment:
I do not quite understand why we need make `ProducerDescriptor` and `TestingShuffleMaster` serializable.
Besides that, I tried run `ExecutionPartitionLifecycleTest` locally with debug log on and do did no see any `NotSerializableException`.

Reply:
It's because the `ShuffleDescriptor` introduced in this `TestShuffleMaster` refers to `ProducerDescriptor` and `ShuffleMaster`. But I shouldn't make them serializable. Sorry for that. Resolved.